### DISCUSSION TITLE
Vanilla JS support

### DISCUSSION THREAD

**Discussion by vitalyaM** [barvian/number-flow#40] [Ideas]

**Comment by barvian** [comment]
Hi! Yes, definitely. All the core logic is in the `number-flow` package which is a custom element. It just needs a better API as right now it was designed for framework wrappers.

**Reply by jamaluddinrumi** [reply]
ow, turns out that so it's already exist 

https://github.com/barvian/number-flow/tree/main/packages/number-flow

kudos

**Comment by charrysong** [comment]
it's great

**Comment by barvian** [comment]
Finally got around to this!

https://number-flow.barvian.me/vanilla
